### PR TITLE
Feature/other locations

### LINKED ISSUE #31: User can select "other" location
User can select "other" from location dropdown.

Users are prompted to type in an alternative location.
We receive this location and store it somewhere.

We need a way to make a judgment at some point in the future as to where to open additional WR chapters, so we need a way to document future location suggestions.

### PR DESCRIPTION
**Pull request by rebeccafae**
## Migrations
YES

## Description
Allow users to choose "Other" location, and when they do, they should provide their zipcode.

## Deploy Notes
Requires db migration

## Github Issue
Fixes #31 

## Background

`lives_in_detroit` is no longer there for the user to choose, but has not been removed from the code base. We need to make a decision whether to add it back, and have both for a time on staging. Or just agree not to deploy staging to production until we change the code base logic. (Should be the very next PR either way)

## Screenshots

image

image

## Definition of Done

- [X] This PR has appropriate test coverage.

### CODE REVIEW

**Comment by rebeccafae** [comment]
@aelydens @westonplatter I have a question above under Background. Will repost in this comment as well.
`lives_in_detroit` is no longer there for the user to choose, but has not been removed from the code base. We need to make a decision whether to add it back to the view and have both for a time on staging. Or just agree not to deploy staging to production until we change the code base logic. (Should be the very next PR either way). Which would you both prefer?


Edit: This question is no longer relevant, as #40 solves the problem

**Comment by rebeccafae** [comment]
@marshallds would you review?

File: db/migrate/20170218190259_add_zip_code_to_user.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddZipCodeToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :zip_code, :integer
```
Comment (marshallds) [inline db/migrate/20170218190259_add_zip_code_to_user.rb:3]: zip codes aren't actually scalars.  Storing as an integer will truncade leading zeros for places like Boston (02108 would be stored as 2108)

**Review by marshallds — CHANGES_REQUESTED** [review]
looks pretty good.  Just one thing...

**Review by marshallds — APPROVED** [review]

**Pull request merged**